lib/tasks/uploads.rake

```diff
@@ -93,8 +93,20 @@ task "uploads:migrate_from_s3" => :environment do
   ENV["RAILS_DB"] ? migrate_from_s3 : migrate_all_from_s3
 end
 
-task "uploads:batch_migrate_from_s3", [:limit] => :environment do |_, args|
-  ENV["RAILS_DB"] ? migrate_from_s3(limit: args[:limit]) : migrate_all_from_s3(limit: args[:limit])
+# limit is the limit to consider migrating, max is the max to actually migrate
+# This is to manage resources during testing; limit 1000 max 1 is a reasonable test
```

Comment:
> limit is the limit to consider migrating, max is the max to actually migrate

I _think_ I know what both of these do by reading the code, but this comment doesn't make it clear. `max` is the **number** of **uploads** to be migrated? And `limit` is used in SQL queries? (does that need to be configurable?)